helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ClusterAccessor.java

```diff
@@ -617,10 +630,6 @@ public Response updateCloudConfig(@PathParam("clusterId") String clusterId,
     }
     try {
       switch (command) {
-      case delete: {
-        admin.removeCloudConfig(clusterId);
-      }
-      break;
       case update: {
         try {
           CloudConfig cloudConfig = new CloudConfig.Builder(record).build();
```

Comment:
Can you have another PR like Molly's work? The update/delete for a cloudConfig is working on update/delete entry of the CloudConfig.

Reply:
I will do new PR for update/delete the entries.